mordant/build.gradle.kts

```diff
@@ -34,8 +34,6 @@ kotlin {
     sourceSets {
         all {
             with(languageSettings) {
-                languageVersion = "1.6"
-                apiVersion = "1.6"
                 optIn("kotlin.RequiresOptIn")
```

Comment:
This can be removed since 1.7: https://kotlinlang.org/docs/whatsnew17.html#stable-opt-in-requirements